StudentList/Domain/Actions/StudentListChanged.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace StudentList.Domain.Actions
+{
+    public class StudentListChanged
```

Comment:
What this action is supposed to indicate?

Reply:
When I add new student or change existing, I handle this action and return null student's collection so it means app should load students from the repository